src/org/zaproxy/zap/extension/websocket/resources/Messages.properties

```diff
@@ -47,14 +47,30 @@ websocket.filter.label.regex.inverse            = Inverse
 websocket.fuzz.fail                             = Error
 websocket.fuzz.success                          = Successful
 websocket.manual_send.fail                      = Unable to send crafted message!
+websocket.manual_send.fail.unable_reopen        = It was not possible to reopen the connection.
 websocket.manual_send.fail.disconnected_channel = Selected WebSocket channel is not connected.
+websocket.manual_send.fail.retrieve             = An error occurred while trying to retrieve the HTTP handshake from history. Please reopen the connection from browser.
 websocket.manual_send.fail.invalid_channel      = Invalid WebSocket channel selected.
 websocket.manual_send.fail.invalid_direction_client_mode = Selected WebSocket channel is not connected to the client.
 websocket.manual_send.fail.invalid_opcode       = Invalid WebSocket opcode selected.
+websocket.manual_send.fail.out_of_scope         = Request out of scope.
+websocket.manual_send.use_reopen                = Use the reopen button to establish a new connection.
+websocket.manual_send.btn_reopen_hint           = Reopen the closed connection (in new channel)
+websocket.manual_send.btn_reopen_edit_hint      = Reopen Configuration
+websocket.manual_send.try_to_reopen             = Connecting...
```

Comment:
There was already `websocket.manual_send.adv_dialog.connecting`, remove that one?